crypto/x509/x_attrib.c

```diff
@@ -279,6 +301,12 @@ int ossl_print_attribute_value(BIO *out,
 
     /* Would it be approriate to just hexdump? */
     default:
-        return BIO_printf(out, "%*s<Unsupported tag %d>", indent, "", av->type);
+        if (BIO_printf(out,
```

Comment:
Too much line-breaks.
No need to put each calling argument on its own line.

Reply:
This just barely does not fit in the 80-column limit, which is why I wrapped it. I will mark this as resolved, but I'd be happy to fix it if you have a suggestion for a solution.